### PR TITLE
html and css files for who is tec page

### LINKED ISSUE #4: Create Who Is Tec Page
**Who is tec page :** 
Is a description page of tec application. 

- [x] Create header section in html and its style . It contains:
- Tec logo.
- Home icon and its back to home page.
- [x] Create content section in html and its style. It contains:
 - Image of tec.
 - Introduction paragraph to tec. 
- [x] Create speak section in html and its style.
- [x] Create button for start in html and its style.
- [x] Make the page responsive

### PR DESCRIPTION
**Pull request by ahmedsinwar**
#4

### CODE REVIEW

File: css/style-who-is-tec.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+p {
+    Font-family : 'Rowdies', cursive;
+    font-style: normal;
```
Comment (AlaaTaima) [inline css/style-who-is-tec.css]: **`normal` is the default value so, you don't need to add it.**

File: css/style-who-is-tec.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    height: 75px;
+    text-align: center;
+    border-radius: 51px;
+    box-shadow: inset 0px 3px 14px rgba(255, 52, 130, 0.33);
+    background-color: #82FF34;
+    margin: 0 85px;
+}
+
+.button-paragraph {
+    font-style: normal;
```
Comment (AlaaTaima) [inline css/style-who-is-tec.css]: **As the previous comment :** 

**`normal` is the default value so, you don't need to add it.**

File: html/who-is-tec.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../css/style-who-is-tec.css">
+    <link href="https://fonts.googleapis.com/css2?family=Rowdies:wght@300;400;700&display=swap" rel="stylesheet">
+    <title>Who Is Tec</title>
+</head>
+<body>
+    <div class="container">
```
Comment (AlaaTaima) [inline html/who-is-tec.html]: **Use the semantic element `main` here instead of `div`.**

File: html/who-is-tec.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+        <!-- Header Section  -->
+
+        <header class="container-header">
+
+            <img src="../Assets/logo.png" alt="logo" class="container-header-logo">
```
Comment (AlaaTaima) [inline html/who-is-tec.html]: **Use the semantic element `figure` with an image here and do the same with the rest**

File: html/who-is-tec.html [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+            <img src="../Assets/robot.png" alt="robot">
+
+            <p class="container-content-introduction">
+                Hi Friends, Iam Tec. Your new English instructor.<br>
+                Iam happy to teach you how to pronounce English<br>
+                words correctly and know their meanings.
+            </p>
+
+        </div>
```
Comment (AlaaTaima) [inline html/who-is-tec.html]: **Use the semantic element `section` here instead of `div`**

File: html/who-is-tec.html [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+        <!-- Speak Section -->
+
+        <div class="container-speak">
+
+            <p class="container-speak-paragraph"> You can call me just Tec </p>
+            <img src="../Assets/volume.png" alt="speak icon" id="volume"> 
+
+        </div>
```
Comment (AlaaTaima) [inline html/who-is-tec.html]: **As the previous comment :** 

**Use the semantic element `section` instead of `div`**

**Review by AlaaTaima — COMMENTED** [review]
**Great job 👍** 

* But there are some points that would be awesome to get enhanced, Follow them at the next comments then, pull main and push again.
* Fix responsive for desktop and tablet.

**Review by ahmadfarid-059 — DISMISSED** [review]

**Review by AlaaTaima — CHANGES_REQUESTED** [review]
@ahmedsinwar , 
* You have duplicated files of html&css of `who-is-tec page ` 
   remove one of them as we agreed on previously.

* You have a folder for images called `images` as we agreed on , so why did you add another one which called assets !!
* Do `git pull origin main` then push again.

**Review by AlaaTaima — APPROVED** [review]

**Pull request merged**